src/context/ThemeContext.js

```diff
@@ -0,0 +1,42 @@
+// References
+// https://github.com/KnightHacks/hackathon-2021-frontend/
+
+import { createContext, useEffect, useState } from "react";  
+
+
+const getInitialTheme = () => {
+  if ((localStorage.getItem("theme") === "dark") || (localStorage.getItem("theme") === null)) {
+    return "dark";
```

Comment:
Return `(window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) ? "dark" : "light"` instead if theme is `null`, unless we want to add an "auto" switch to sync with OS settings.